src/services/BidService.ts

```diff
@@ -160,4 +160,12 @@ export default class BidService {
       publish_at: pendingBids[0]?.publish_at,
     }
   }
+
+  static async getOpenTendersTotal() {
+    const data = await UnpublishedBidModel.getOpenTendersTotal()
+
```

Comment:
Total tenders in the bids model? Shouldn't this be in the proposals model? Maybe it is the name that confuses me

Reply:
But it's querying the bids table 🤔